docs/About.md

```diff
@@ -0,0 +1,21 @@
+# About
+
+CYNQ is an alternative framework to code applications for Xilinx FPGAs with the easiness of PYNQ. Currently, most of the flow is quite complex and requires Vitis to get a decent C++ application and there are no bindings from the official PYNQ. Based on these needs, CYNQ proposes:
```

Comment:
replace easiness by pynq like api, the word keeps popping up, also change CYNQ proposes by "Cynq has the following objectives:"